router.ts

```diff
@@ -0,0 +1,21 @@
+import Router = require('koa-router');
+
+const router = new Router();
```

Comment:
what if you will have another routers? is `router` a good name?

Reply:
done